frontend/src/libs/components/header/styles.module.scss

```diff
@@ -88,6 +88,11 @@ img.avatar:hover {
     display: none;
   }
 
+  .header {
+    height: 80px;
```

Comment:
Not sure about overwriting header height this way - as above it's defined through variable - won't it impact other places dependent on header height?